client/src/main/scala/skuber/api/Configuration.scala

```diff
@@ -228,6 +231,19 @@ object Configuration {
       }
     }
 
+  private lazy val inClusterConfigReloadInterval: FiniteDuration = {
+    import scala.concurrent.duration._
+    // We default to 10 minutes because modern Kubernetes distributions issue service account tokens that are valid for
+    // one hour, rotating them when they reach 80% of their lifespan. This means, any given arbitrary read of the token
+    // from the filesystem can only be sure that that token will be valid for a maximum of 12 minutes (20% of 1 hour).
+    // This is configurable, hence we allow it to be overridden by an environment variable.
+    // Note that the equivalent Go code for this reloads every minute, erroneously stating in a comment that tokens are
+    // rotated every 10 minutes. See https://github.com/kubernetes/client-go/blob/4f9edc15a7e71c3f9c7874a872a2545c8737726c/transport/token_source.go#L75-L79
+    sys.env.get("SKUBER_TOKEN_RELOAD_INTERVAL_SECONDS")
```

Comment:
Can we explicitly check that if the value is zero or less and if so then disable the token reloading? This is in case there is some edge case where token reloading is problematic so users can restore the old behaviour.